### DISCUSSION TITLE
How can I render/compile passing a file path as an argument?

### DISCUSSION THREAD

**Discussion by salemalem** [eta-dev/eta#144] [Q&A]
How can I render/compile passing a file path as an argument?
Usually I did this.response.render("index") like here https://github.com/salemalem/deno-twd-drash/blob/main/drash-resources/home_resource.js
Now, I want to render/compile and save the output as .HTML to ship to dist so users can host static files without a server.

**Comment by salemalem** [comment]
Check this https://github.com/salemalem/deno-twd-drash/blob/661edbaeff08abbb76b66655a4db31a0822e0d79/drash-resources/saving_render_as_html_resource.js to see how I saved output as html to render later. This is rendered HTML output https://github.com/salemalem/deno-twd-drash/blob/661edbaeff08abbb76b66655a4db31a0822e0d79/generated.html

**Comment by shadowtime2000** [comment]
@salemalem Can you clarify your question? If your asking how to render files you can use the `renderFile` function which will render a template from a file. [Example for Deno](https://eta.js.org/docs/examples/deno).
